plugins/Tools/RotateTool/RotateTool.py

```diff
@@ -264,6 +268,23 @@ def setRotationSnapAngle(self, angle):
             self._snap_angle = angle
             self.propertyChanged.emit()
 
+    ##  Wether the rotate tool is in 'Lay flat by face'-Mode.
+    #
+    #   \return (bool)
+    def getSelectFaceToLayFlatMode(self):
+        if not Selection.getFaceSelectMode():
+            self._selectFaceMode = False  # .. but not the other way around!
+        return self._selectFaceMode
+
+    ##  Set the rotate tool to/from 'Lay flat by face'-Mode.
+    #
+    #   \param (bool)
+    def setSelectFaceToLayFlatMode(self, select):
```

Comment:
Missing mypy typing

Reply:
Then we should do the typing of all the tools :-)

Reply:
We've previously said that we'd add this typing boy-scout-style. When you edit something close by, add typing.

Reply:
I think I remember* trying this, and then it gave more and more warnings (which would've tripped up the CI as well). So I added more typing, repeat. Eventually I had to conclude that it went way out of scope of the changes.

*) I'm not entirely sure it was for _this_ PR though.